src/event_loop.rs

```diff
@@ -189,10 +189,22 @@ fn handle_event(
             app.table.next();
             app.on_select();
           }
+          KeyCode::Char('n') => {
```

Comment:
we'll need to include the `event.modifiers == KeyModifiers::CONTROL` part in the pattern itself so that when you press `n` or `p` without ctrl being held, we fall through to:
```
KeyCode::Char(c) => {
  handle_keybinding_press(app, loading_tx, tx, c);
}
```

Admittedly I'm actually not sure how to do that

Reply:
```rust
          KeyCode::Char('n') if event.modifiers == KeyModifiers::CONTROL => {
              app.table.next();
              app.on_select();
          }
```

this seems to work

Reply:
looks like this is called `Match guards`
https://doc.rust-lang.org/reference/expressions/match-expr.html#match-guards

Reply:
what u reckon?

Reply:
that looks good: are we able to combine that with the other pattern for selecting the next item? That way we only need to update one place if we change the block

Reply:
doesn't seem to be possible, the Match guard is applied to all the matches :(  

A possible approach is abstract the navigation "step" to a function and those matches only call that function:
something like:

```rust
fn nagivate_down() {
  app.table.next();
  app.on_select();
}

...
KeyCode::Down | KeyCode::Char('j') => { 
  nagivate_down()
}
KeyCode::Char('n') if event.modifiers == KeyModifiers::CONTROL => { 
  nagivate_down()
}
```

any thoughts?

Reply:
just for reference:

https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html#extra-conditionals-with-match-guards

![image](https://user-images.githubusercontent.com/10784051/109447355-33a07d80-7a8f-11eb-8dff-cbe747214503.png)

Reply:
yeah that works :) Shame that match guards always apply to the whole pattern